examples/pytorch/answer-generator/predictor.py

```diff
@@ -0,0 +1,29 @@
+import wget
+import torch
+from transformers import GPT2Tokenizer, GPT2LMHeadModel, GPT2Config
+import generator
+
+
+medium_config = GPT2Config(n_embd=1024, n_layer=24, n_head=16)
+model = GPT2LMHeadModel(medium_config)
+tokenizer = GPT2Tokenizer.from_pretrained("gpt2")
+
+
+def init(model_path, metadata):
+    wget.download(
+        "https://convaisharables.blob.core.windows.net/lsp/multiref/medium_ft.pkl", "medium_ft.pkl"
+    )
+
+    weights = torch.load("medium_ft.pkl")
+    weights["lm_head.weight"] = weights["lm_head.decoder.weight"]
+    weights.pop("lm_head.decoder.weight", None)
+
+    model.load_state_dict(weights)
```

Comment:
`model.load_state_dict({'lm_head.weight': weights['lm_head.decoder.weight']})` defining a new dictionary is preferred over  popping a key from a dictionary